requirements.optional.txt

```diff
@@ -22,7 +22,11 @@ pyneuroml >= 0.5.18
 python_libsbml
 
 [smoldyn]
-smoldyn >= 2.66
+smoldyn # >= 2.70
+# simulariumio
```

Comment:
comment - remove comment?

Reply:
The commented-out simulariumio requirement will eventually become one of the requirements that come with the optional [smoldyn] install, i.e: any spatial simulator.